src/maxmind.cpp

```diff
@@ -103,6 +140,10 @@ std::vector < std::string > maxmind_bindings::connection(MMDB_s *data, std::vect
   return mmdb_getstring(data, ip_addresses, "connection_type", NULL);
 }
 
+std::vector < int > maxmind_bindings::city_geoname_id(MMDB_s *data, std::vector < std::string >& ip_addresses){
+  return mmdb_getint(data, ip_addresses, "city", "geoname_id", NULL);
```

Comment:
Actually looks like that's a std::string problem, probably because the underlying representation of a String SEXP is a char. Bloody R. So this'll be fine.